ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_latin.txt

```diff
@@ -235,53 +240,63 @@
+			gens_Postumia gens_Pompeia gens_Rabiria gens_Quinctia gens_Salvia gens_Scribonia gens_Sempronia gens_Sertoria gens_Silia gens_Titia
+			gens_Tullia gens_Ulpia gens_Valeria gens_Vesuvia gens_Vinicia gens_Viridia gens_Vitellia
 		}
 
 		male_names = {
-			Aemilian Aemilius Aetius Albinus Alexander Anastasius Anicius Annius Antoninus Antonius Arcadius Aurelian Aurelius
-			Avitus Belisarius Britannicus Caecilius Caesar Carinus Carus Cassius Claudius Clementius Constans Constantine
-			Constantius Cornelius Crassus Decimus Decius Diocletian Domitian Domitius Drusus Ennodius Fabianus Fabius Faustus Felix Flavianus Florian
-			Gaius Galerius Gallienus Gallus Germanus Gordian Gracchus Gratian Hadrian Herculius Honorius Joannes Jovian
-			Julian Julius Julius-Caesar Junius Justin Justinian Leo Leontius Liberius Licinius Lucius Magnus Marcellinus
-			Marcellus Marcian Marcus Marinus Marius Martinus Maximian Maximinus Maximus Nepotian Nerva Numerian Octavianus Octavius Otho
-			Paulinus Paulus Petronius Philip Pompeius Pontianus Priscus Probus Publius Quintus Romanus Romulus Rufinus Rutilius Sabinianus
-			Saturninus Sempronius Septimius Sergius Severinus Severus Sextus Symmachus Tacitus Theodosius Tiberius
-			Titius Titus Trajan Urbanus Valens Valentinian Valerian Valerius Venantius Vespasian Victor Victorinus Vigilius Vitalianus Volusianus
+			Gaius Lucius Marcus Publius Quintus Titus Tiberius Sextus
+			Aulus Decimus Gnaeus Spurius Manius Servius Appius Numerius
+			Vibius
+			Amulius Camillus Decius Drusus Faustus Flavius Gallus
+			Herius Hostus Opiter Octavius Postumus Primus Proculus
+			Secundus Tertius Ancus
 		}
 		female_names = {
-			Aelia Agrippina Antonia Antonina Aquilia Aurelia Caecilia Caesaris Candida Cassia Claudia Clementia Constantia
-			Constantina Cornelia Crispina Dominica Domitiana Domna Domnina Drusilla Fausta Faustina Flacilla Flavia Fortunata Frontina Gaia Galeria
-			Gordiana Hadriana Helvia Hilara Honoria Hortensia Julia Juliana Junia Justa Justina Justiniana Licinia Livia Livilla Lucilla
-			Magnia Magnilla Marcella Marcia Marciana Marina Martina Matrona Maxima Maximiana Messalina Minervina Minucia Modia Mustela
-			Octavia Palatina Paula Paulina Petronella Petronia Placidia Plautia Plautilla Pompeia Poppaea Praeiecta Publia Pulcheria
-			Rusticiana Sabina Sallustia Salonina Salvianella Scantilla Serena Sergia Severa Severina Silvia Spadusa Sulpicia Tarsilla Titiana Ulpia Valeria
-			Verina Victoria Vigilantia Viviana
+			Gaia Lucia Marca Publia Quinta Tita Tiberia Sexta Aula Decima
+			Gnaea Spuria Mania Servia Appia Numeria Vibia
+			Amulia Camilla Decia Drusa Fausta Flavia Galla Heria
+			Hosta Opitera Octavia Postuma Prima Procula Secunda Tertia Anca
 		}
-		dynasty_of_location_prefix = "dynnp_of"
-		patronym_prefix_male = "dynnpat_pre_filius"
+		
+		dynasty_of_location_prefix = "dynnp_a"
```

Comment:
What is the prefix based on?

Reply:
If it were possible to use suffixes, I'd have a place called _Gießen_ and have _Gießenensis_. But since this can't be, _a Gießen_ ("from Gießen") is a valid enough alternative. _A(b)_ as a preposition is sometimes used in Latin to translate the _de X_ and the _von X_.